src/features/movies/MoviePage/People/styled.js

```diff
@@ -1,26 +1,12 @@
 import styled from "styled-components";
 
-export const Wrapper = styled.section`
-    margin-top: 64px;
-`;
-
-export const Header = styled.header`
-    font-weight: 600;
-    font-size: 36px;
-    line-height: 120%;
-    margin: 0 0 32px;
-`;
-
 export const List = styled.ul`
```

Comment:
This List styled-component is not used anywhere. Actually we dont need this file at all now.

Reply:
you right i missed this too, i deleted this styled file